src/Cli/dotnet/commands/dotnet-tool/install/ToolInstallGlobalOrToolPathCommand.cs

```diff
@@ -51,6 +51,8 @@ internal class ToolInstallGlobalOrToolPathCommand : CommandBase
         private readonly string _currentWorkingDirectory;
         private readonly bool? _verifySignatures;
 
+        internal readonly RestoreActionConfig restoreActionConfig;
```

Comment:
I see this was made `internal` so that it could be accessed via tests.
@dsplaisted What do you think of this pattern? I am not opposed to it, but Im not sure if we want this to be available in the assembly. Are people likely to start using this if we do that? If you don't have a strong opinion I want to ask @/rainersigwald 

If we decide against this, you could also test via a pattern like how  `WhenRunWithArchOptionItDownloadsAppHostTemplate` works, I believe.

Reply:
This is fine, IMHO, I'm not too worried about us accidentally starting to abuse this in our code.